cmd/zc_traverser_local.go

```diff
@@ -217,7 +217,7 @@ func WalkWithSymlinks(appCtx context.Context, fullPath string, walkFunc filepath
 	// do NOT put fullPath: true into the map at this time, because we want to match the semantics of filepath.Walk, where the walkfunc is called for the root
 	// When following symlinks, our current implementation tracks folders and files.  Which may consume GB's of RAM when there are 10s of millions of files.
 	var seenPaths seenPathsRecorder = &nullSeenPathsRecorder{} // uses no RAM
-	if followSymlinks {
+	if symlinkHandling.Follow() {                              // only if we're following we need to worry about this
```

Comment:
In case of broken symlinks do we log when we try to follow it?

Reply:
For failed symlink traversals, yes.